src/Microsoft.SqlTools.ServiceLayer/NotebookConvert/Contracts/NotebookConvertContracts.cs

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.NotebookConvert.Contracts
+{
+    public class ConvertNotebookToSqlParams : GeneralRequestDetails
+    {
+        /// <summary>
+        /// The raw Notebook JSON content to convert
+        /// </summary>
+        public string NotebookJson { get; set; }
+
+    }
+
+    public class ConvertNotebookToSqlResult : ResultStatus
+    {
+        /// <summary>
+        /// The raw SQL query content to display
+        /// </summary>
+        public string content;
+    }
+
+    public class ConvertNotebookToSqlRequest
+    {
+        public static readonly
+            RequestType<ConvertNotebookToSqlParams, ConvertNotebookToSqlResult> Type =
+            RequestType<ConvertNotebookToSqlParams, ConvertNotebookToSqlResult>.Create("notebookconvert/convertnotebooktosql");
+    }
+
+    public class ConvertSqlToNotebookParams : GeneralRequestDetails
+    {
+        /// <summary>
+        /// The ClientUri of the SQL Query file we're converting
+        /// </summary>
+        public string ClientUri { get; set; }
```

Comment:
I think we typically call it ownerUri.

Reply:
For some reason I decided to call it ClientUri in STS..although I can't remember why exactly. Mostly cause it came from the client I guess? The use here is even more incorrect anyways - the method I'm calling (GetFile) has the parameter as filePath but it takes in URI's too... 

https://github.com/microsoft/sqltoolsservice/blob/main/src/Microsoft.SqlTools.ServiceLayer/Utility/ResolvedFile.cs#L25

Anyways I agree we should settle on something here - but in this case I wanted to stick with ClientUri for consistencies sake until we could look at how these are used across the app in its entirety.